tests/integration_test.py

```diff
@@ -246,30 +247,80 @@ def runTest(self):
         self.assertFalse(inspect_data['VolumesRW'][mount_dest])
 
 
-@unittest.skipIf(NOT_ON_HOST, 'Tests running inside a container; no syslog')
-class TestCreateContainerWithLogConfig(BaseTestCase):
-    def runTest(self):
-        config = docker.utils.LogConfig(
-            type=docker.utils.LogConfig.types.SYSLOG,
-            config={'key1': 'val1'}
+class CreateContainerWithLogConfigTest(BaseTestCase):
```

Comment:
This test is written in a convention that echoes that of docker-compose, https://github.com/docker/compose/blob/master/tests/unit/config_test.py, which is a good guide of how tests are to be structured and written.

Test case names are easier to read without Test at the beginning, we know it's a test, so we can put it at the end. Naming our tests individually rather than runTest provides helpful feedback when a test fails or we get a stack trace error. This is a way of improving the code while I was there and a demonstration of what the tests can be refactored towards. 

Consistency for it's own sake forsaking that of improved code is not as beneficial.